tests/Feature/ProviderPrivateNetworkTest.php

```
@@ -0,0 +1,493 @@
+        ]);
+
+        /** @var Hetzner $provider */
+        $provider = $this->hetznerConnection()->provider();
+
+        $networks = $provider->privateNetworks(['101'], []);
+
+        $this->assertNull(
+            $networks[0]->members[0]->ip,
+            'Member addresses reach an unquoted shell interpolation in the ufw template.'
+        );
+    }
```

PHPMD (2.15.0)

[error] The method test_member_addresses_that_are_not_plain_ipv4_are_dropped is not named in camelCase. (undefined)

(CamelCaseMethodName)